rover-server/internal/mapper/rover/in/exposure.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2025 Deutsche Telekom IT GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package in
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	roverv1 "github.com/telekom/controlplane/rover/api/v1"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+
+	"github.com/telekom/controlplane/rover-server/internal/api"
+)
+
+func mapExposure(in *api.Exposure, out *roverv1.Exposure) error {
+	expType, err := in.Discriminator()
+	if err != nil {
+		return errors.Wrap(err, "failed to get exposure type")
+	}
+	switch expType {
+	case "api":
+		apiExp, err := in.AsApiExposure()
+		if err != nil {
+			return errors.Wrap(err, "failed to convert to ApiExposure")
+		}
+
+		out.Api = mapApiExposure(apiExp)
+
+	case "event":
+		eventExp, err := in.AsEventExposure()
+		if err != nil {
+			return errors.Wrap(err, "failed to convert to EventExposure")
+		}
+
+		out.Event = &roverv1.EventExposure{
+			EventType: eventExp.EventType,
+		}
+
+	default:
+		return errors.Errorf("unknown exposure type: %s", expType)
+	}
+
+	return nil
+}
+
+func mapApiExposure(in api.ApiExposure) *roverv1.ApiExposure {
+	out := &roverv1.ApiExposure{}
+	out.BasePath = in.BasePath
+	out.Visibility = toRoverVisibility(in.Visibility)
+	out.Approval = roverv1.Approval{
+		Strategy: toRoverApprovalStrategy(in.Approval),
```

Comment:
trustedTeams are not transferred: https://github.com/telekom/controlplane/blob/feat/rover-server/rover/api/v1/rover_types.go#L309